spec/current_players_spec.rb

```diff
@@ -48,5 +48,15 @@ def make_a_player(marker)
 
             expect(new_board_spaces).to eq ["", "", "", "", "", "", "", "", "O"]
         end
+
+        def handle_turn(marker)
```

Comment:
Does this helper handle a turn? It looks to me like it simply returns a new player with a given marker